yaml/csr.yaml

```diff
@@ -1203,200 +1211,269 @@
       reset_val: 0
       msb: 18
       lsb: 18
+      warl_legalize: |
+        val_out = 0
     - field_name: R
       description: >
         Reserved
       type: WARL
       reset_val: 0
       msb: 17
       lsb: 17
+      warl_legalize: |
+        val_out = 0
     - field_name: Q
       description: >
         Quad precision floating point
       type: WARL
       reset_val: 0
       msb: 16
       lsb: 16
+      warl_legalize: |
+        val_out = 0
     - field_name: P
       description: >
         Packed SIMD
       type: WARL
       reset_val: 0
       msb: 15
       lsb: 15
+      warl_legalize: |
+        val_out = 0
     - field_name: O
       description: >
         Reserved
       type: WARL
       reset_val: 0
       msb: 14
       lsb: 14
+      warl_legalize: |
+        val_out = 0
     - field_name: N
       description: >
         User mode interrupts
       type: WARL
       reset_val: 0
       msb: 13
       lsb: 13
+      warl_legalize: |
+        val_out = 0
     - field_name: M
       description: >
         Integer multiply and divide
       type: WARL
-      reset_val: unknown
       msb: 12
       lsb: 12
+### COND M_EXT
+      reset_val: 1
+      warl_legalize: |
+        val_out = 1
+### ENDCOND
+### COND M_NONE
+      reset_val: 0
+      warl_legalize: |
+        val_out = 0
+### ENDCOND
     - field_name: L
       description: >
         Tentatively reserved for decimal floating point extension
       type: WARL
       reset_val: 0
       msb: 11
       lsb: 11
+      warl_legalize: |
+        val_out = 0
     - field_name: K
       description: >
         Reserved
       type: WARL
       reset_val: 0
       msb: 10
       lsb: 10
+      warl_legalize: |
+        val_out = 0
     - field_name: J
       description: >
         Tentatively reserved for dynamically translated languages extension
       type: WARL
       reset_val: 0
       msb: 9
       lsb: 9
+      warl_legalize: |
+        val_out = 0
     - field_name: I
       description: >
         RV32I/RV64I/RV128I ISA
       type: WARL
-      reset_val: unknown
       msb: 8
       lsb: 8
+### COND I_EXT
+      reset_val: 1
+      warl_legalize: |
+        val_out = 1
+### ENDCOND
+### COND E_EXT
+      reset_val: 0
+      warl_legalize: |
+        val_out = 0
+### ENDCOND
     - field_name: H
       description: >
         Hypervisor extension
       type: WARL
       reset_val: 0
       msb: 7
       lsb: 7
+      warl_legalize: |
+        val_out = 0
     - field_name: G
       description: >
         Additional standard extensions present
       type: WARL
       reset_val: 0
       msb: 6
       lsb: 6
+      warl_legalize: |
+        val_out = 0
     - field_name: F
       description: >
         Single presition floating point
       type: WARL
       reset_val: 0
       msb: 5
       lsb: 5
+      warl_legalize: |
+        val_out = 0
     - field_name: E
       description: >
         RV32E base ISA
       type: WARL
-      reset_val: unknown
```

Comment:
"unknown" fails to convert to an integer when using this file to generate the csr access test.